app/views/users/_mooc_provider_settings.html.slim

```diff
@@ -9,6 +9,7 @@ h2
           .provider
             a data-toggle="collapse" href="#collapse-#{provider[:id]}"
               img src = @provider_logos[provider[:logo_id]]
+              = " #{provider[:url]}"
```

Comment:
Is there a simple solution to prevent the 'space' from being underlines when you hover it?